modules/lvms-creating-an-logical-volume-manager-cluster.adoc

```diff
@@ -4,78 +4,203 @@
 
 :_content-type: PROCEDURE
 [id="lvms-creating-lvms-cluster_{context}"]
-= Creating Logical Volume Manager cluster
+= Creating a Logical Volume Manager cluster on a {sno} worker node
 
-You can create a Logical Volume Manager cluster after you install {lvms}.
+You can configure a {sno} worker node as a Logical Volume Manager cluster.
+On the control-plane {sno} node, {lvms} detects and uses the additional worker nodes when the new nodes show up.
 
-{product-title} supports additional worker nodes for {sno} clusters on bare-metal user-provisioned infrastructure.
-{lvms} detects and uses the additional worker nodes when the new nodes show up.
-In case you need to set a node filter for the additional worker nodes, you can use the YAML view while creating the cluster.
+[NOTE]
+====
+When you create a Logical Volume Manager cluster, `StorageClass` and `LVMVolumeGroup` resources work together to provide dynamic provisioning of storage.
+`StorageClass` CRs define the properties of the storage that you can dynamically provision.
+`LVMVolumeGroup` is a specific type of persistent volume (PV) that is backed by an LVM Volume Group.
+`LVMVolumeGroup` CRs provide the backend storage for the persistent volumes that you create.
```

Comment:
IBM Style suggests "back-end" (usually hyphenated) - I found both hyphenated and non-hyphenated versions in the OCP docs repo - more hyphenated than not, so I'm not sure about this particular usage. 
```suggestion
`LVMVolumeGroup` CRs provide the back-end storage for the persistent volumes that you create.
```